src/app/services/playlists/PlaylistService.ts

```diff
@@ -0,0 +1,29 @@
+import {injectable, inject} from 'inversify'
+import {PlaylistDao} from './PlaylistDao'
+import {Playlist} from './Playlist.schema'
+import {Dao} from '../common/BaseDao'
+import {TYPES} from '../../dependencies/types'
+import {CreatePlaylistDTO} from '../../modules/playlists/dtos/CreatePlaylistDTO'
+
+@injectable()
+export class PlaylistService {
+  @inject(TYPES.PlaylistDao)
+  private dao: Dao<Playlist>
+
+  /**
+   * @param {CreatePlaylistDTO} createPlaylistDTO
+   * @return {Promise<Playlist>}
+   */
+  async create(createPlaylistDTO: CreatePlaylistDTO) {
+    return this.dao.create({
+      name: createPlaylistDTO.name
+    })
+  }
+
+  /**
+   * @return {Promise<Playlist[]>}
+   */
+  async get() {
```

Comment:
I guess we can do return types now? Something like `async get() : Promise<Playlist[]> {`?